src/radar-chart/index.js

```diff
@@ -173,6 +174,74 @@ function getPolygons(props) {
   });
 }
 
+/**
+ * Generate circles at the polygon points for Hover functionality
+ * @param {Object} props
+ - props.animation {Boolean}
+ - props.data {Array} array of object specifying what values are to be plotted
+ - props.domains {Array} array of object specifying the way each axis is to be plotted
+ - props.startingAngle {number} the initial angle offset
+ - props.style {object} style object for the whole chart
+ - props.onValueMouseOver {function} function to call on mouse over a polygon point
+ - props.onValueMouseOver {function} function to call when mouse leaves a polygon point
+ * @return {Array} the plotted axis components
+ */
+function getPolygonPoints(props) {
+  const {
+    animation,
+    domains,
+    data,
+    startingAngle,
+    style,
+    onValueMouseOver,
+    onValueMouseOut
+  } = props;
+
+  if (!onValueMouseOver) {
+    return;
+  }
+  const scales = domains.reduce((acc, {domain, name}) => {
+    acc[name] = scaleLinear()
+      .domain(domain)
+      .range([0, 1]);
+    return acc;
+  }, {});
+
+  return data.map((row, rowIndex) => {
+    const mappedData = domains.map(({name, getValue}, index) => {
+      const dataPoint = getValue ? getValue(row) : row[name];
+      // error handling if point doesn't exist
+      const angle = index / domains.length * Math.PI * 2 + startingAngle;
+      // dont let the radius become negative
+      const radius = Math.max(scales[name](dataPoint), 0);
+      return {
+        x: radius * Math.cos(angle),
+        y: radius * Math.sin(angle),
+        domain: name,
+        value: dataPoint,
+        dataName: row.name
+      };
+    });
+
+    return (
```

Comment:
so instead of mapping all of your points to customSVGSeries here you'd would just map them to an array and then use that as the input to a mark series or voronoi

Reply:
I looked into the voronoi for a little bit, and I agree it would be cool to add here (but I didn't add it)

Reply:
Fair this PR has already gotten quite large